api/apps/services/dataset_api_service.py

```
@@ -3921,6 +3942,178 @@ async def _search_layers_all(tenant_id, dataset_id, query, top_k, embd_mdl, kb,
+                select_fields=["name", "parent_kwd"],
+                highlight_fields=[],
+                condition={"compile_kwd": [_NAV_COMPILE_KWD], "type_kwd": ["nav_doc"], name_field: nav_doc_names},
+                match_expressions=[],
+                order_by=OrderByExpr(),
+                offset=0,
+                limit=len(nav_doc_names),
+                index_names=index_nm,
+                knowledgebase_ids=[dataset_id],
+            )
+            field_map = settings.docStoreConn.get_fields(res, ["name", "parent_kwd"])
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)